### ISSUE TITLE
Why not include *.py in sphinx Makefile?

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1460]
When building latex, sphinx includes a Makefile in the build folder. It holds a line for clean-up, such as:

```
#!bash

clean:
    rm -f *.dvi *.log *.ind *.aux *.toc *.syn *.idx *.out *.ilg *.pla
```

However, when I use `make clean` I cannot `make all-pdf` before deleting the PDF generated before.

Why is *.pdf not included in the `Makefile`?

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1460
- Originally reported by: Anonymous
- Originally created at: 2014-05-01T14:25:37.561

**Comment by tk0miya** [comment]
MEMO: This will be fixed in PR #2216.

**Comment by tk0miya** [comment]
Now, #2216 is merged into master branch.
It will be released in next major version.

Thanks,
